### ISSUE TITLE
comparison with cmakelang?

### ISSUE THREAD

**Issue by renefritze** [marzer/check-cmake#3]
Hi @marzer, I happily stumbled over this project in a PyPi search.
I'm currently using `[cmakelang](https://github.com/cheshirekow/cmake_format)` (which hasn't been updated for a while) for my CMake linting needs. Are you aware of that project? 
If yes, can you say anything on feature comparison?
Obviously you're not also targetting formatting. From a quick glance at their [checks](https://github.com/cheshirekow/cmake_format/blob/master/cmakelang/lint/lintdb.py) you seem to be mostly covering other areas?

**Comment by marzer** [comment]
Hi there, good question :) No, I was not familiar with that project, but from a cursory glance it appears as though this one is similar to the `cmake-lint` subproject within it, albeit somewhat less mature.

I can't comment too much on the feature comparsion, other than to suggest you give both a try. Mine is largely focused on finding overtly "bad CMake" and telling you how to fix/modernize (with a keen focus on finding non-target-based bits and nudging you towards targets instead).

The story behind `check-cmake` is: I started a new job and found myself needing to fix the CMake for an entire department who'd all gotten it wrong, so it's calibrated towards that use-case.

**Comment by renefritze** [comment]
Gotcha. Thanks for the info.
I will definitely try this out too. Might be a while though. 
Would you generally be open to getting some contributions porting over checks?

**Comment by marzer** [comment]
> Would you generally be open to getting some contributions porting over checks?

Yep, sure, though be mindful that check-cmake is pretty coarse. It doesn't do a full CMake parse, expand variables, et cetera. It's mostly regex-based. Not all checks will work in that context.
